dev/sort_qmake_file_entries.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
```

Comment:
The script do not use any kind of division, so I think the `division` module should not change anything. `unicode_literals` may be nice, but I would say now it's too late to change :wink: